torchaudio/__init__.py

```diff
@@ -25,6 +25,7 @@
     'compliance',
     'datasets',
     'functional',
+    'models',
```

Comment:
Don't we need to add the actual import statement at L3?

Reply:
Yep, sorry I ignored it. Now it's added.